pkg/cloudmap/utils.go

```diff
@@ -0,0 +1,34 @@
+package cloudmap
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+func IsPodReady(pod *corev1.Pod) bool {
```

Comment:
let's rename it as "IsPodContainersReady(pod *corev1.Pod) bool and check corev1.ContainersReady condition instead.
corev1.PodReady won't be true once we add podReadinessGate. (it will only be true when our podReadinessGate condition be true, however we need to add the pod into cloudmap for our readinessgate pass)

Reply:
ok